infrastructure/lib/monitor.ts

```diff
@@ -0,0 +1,53 @@
+import {Construct, Duration} from '@aws-cdk/core';
+import {Rule, Schedule} from '@aws-cdk/aws-events';
+import {AssetCode, Function, Runtime} from '@aws-cdk/aws-lambda';
+import {join} from 'path';
+import {LambdaFunction} from '@aws-cdk/aws-events-targets';
+import {Table} from '@aws-cdk/aws-dynamodb';
+import { PolicyStatement } from '@aws-cdk/aws-iam';
+
+/**
+ * The required properties for periodically updating a set of monitors
+ * every minute.
+ */
+export interface MonitorCheckProps {
+  /**
+   * The name of the table hosting the monitors to check
+   */
+  table: Table;
```

Comment:
prefix with `readonly` (that's a jsii requirement in case we want to publish this through jsii).